app/screens/settings-screen/settings/account-static-qr.tsx

```diff
@@ -10,21 +10,19 @@ import { SettingsRow } from "../row"
 export const AccountStaticQR: React.FC = () => {
   const { appConfig } = useAppConfig()
   const posUrl = appConfig.galoyInstance.posUrl
-  const shortUrl = appConfig.galoyInstance.lnAddressHostname
 
   const { LL } = useI18nContext()
 
   const { data, loading } = useSettingsScreenQuery()
   if (!data?.me?.username) return <></>
 
   const qrUrl = `${posUrl}/${data.me.username}/print`
-  const short = `${shortUrl}/${data.me.username}/print`
 
   return (
     <SettingsRow
       loading={loading}
       title={LL.SettingsScreen.staticQr()}
-      subtitle={short}
+      subtitle={posUrl}
```

Comment:
same here

Reply:
Now matching the correct url used when opening the link.